ast/modify.go

```diff
@@ -47,6 +47,33 @@ func InjectField(f File, fieldName, fieldValue, sectionName, subsectionName stri
 	return f
 }
 
+// AppendBlankLine appends an empty line to the file.
+func AppendBlankLineToFile(f File) File {
+	f.CommentsAfter = append(f.CommentsAfter, &Comment{})
+	return f
+}
+
+// AppendBlankLineToSection appends an empty to a section
+func AppendBlankLineToSection(f File, sectionName, subsectionName string) (File, bool) {
```

Comment:
Outside of this PR, but it would be nice to make these functions methods of the object they are changing - i.e. `someSection.AppendBlankLine()`

Reply:
Ideally they should be closest to where they are operating on. There are at least two structures that exist: `File` and `Section`, so you could have things like `file.AppendLine()` and `section.AppendLine()`